diff --git a/node_modules/graphile-build/dist/newWithHooks/index.js b/node_modules/graphile-build/dist/newWithHooks/index.js
index dc0d978..46e0a9c 100644
--- a/node_modules/graphile-build/dist/newWithHooks/index.js
+++ b/node_modules/graphile-build/dist/newWithHooks/index.js
@@ -5,6 +5,7 @@ const grafast_1 = require("grafast");
 const graphql_1 = require("grafast/graphql");
 const util_1 = require("util");
 const utils_js_1 = require("../utils.js");
+const { assertName } = require("graphql");
 const isString = (str) => typeof str === "string";
 const knownTypes = [
     graphql_1.GraphQLSchema,
@@ -81,6 +82,11 @@ function makeNewWithHooks({ builder }) {
                             const processedFields = [];
                             const fieldWithHooks = (fieldScope, fieldSpec) => {
                                 const { fieldName } = fieldScope;
+                                try {
+                                assertName(fieldName)
+                                } catch (e) {
+                                    throw new Error(`Attempted to add field ${JSON.stringify(fieldName)} to object type ${Self.name}, but that's not a valid name: ${e.message}`)
+                                }
                                 build.extend(fieldScope, scope, "Adding the object type scope to the field's scope");
                                 if (!isString(fieldName)) {
                                     throw new Error("It looks like you forgot to pass the fieldName to `fieldWithHooks`, we're sorry this is current necessary.");
